File: HarvesterBack/Harvester/Harvester.Domain/Models/Field.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using NetTopologySuite.Geometries;

namespace Harvester.Domain.Models
{
    public class Field
    {
        public int Id { get; set; }
        public string IdentifierName { get; set; }

        public string? CommonName { get; set; }

        public decimal AreaHectares { get; set; }
        public decimal TerrainCoeff { get; set; } = 1.0m;
        public decimal ShapeCoeff { get; set; } = 1.0m;
        public string CropType { get; set; }

        public Point? CenterPoint { get; set; } = default!;

        public MultiPolygon? Boundary { get; set; } = default!;

        public ICollection<Order>? Orders { get; set; }

      
    }
}

File: HarvesterFront/src/app/core/models/Field.ts
import { GeoMultiPolygonDto } from './geoJson/geo-multiPolygon-dto';
import { GeoPointDto } from './geoJson/geo-point-dto';

export interface Field {
  id: number;
  identifierName: string;
  commonName: string;
  areaHectares: number;
  terrainCoeff: number;
  shapeCoeff: number;
  cropType: string;
  centerPoint: GeoPointDto;
  boundary: GeoMultiPolygonDto;
}
